### PR TITLE
fix: make it possible to translate all xdrip preferences

### PR DESCRIPTION
**Pull request by k2s**

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java [hunk 1]
```diff
@@ -2792,7 +2792,7 @@ private void displayCurrentInfo() {
         if (sensor_age > 0 && (DexCollectionType.hasLibre() || hasLibreblock())) {
             final String age_problem = (Pref.getBooleanDefaultFalse("nfc_age_problem") ? " \u26A0\u26A0\u26A0" : "");
             if (Pref.getBoolean("nfc_show_age", true)) {
-                sensorAge.setText(getResources().getQuantityString(R.plurals.sensor_age, sensor_age, JoH.qs(((double) sensor_age) / 1440, 1)) + age_problem);
+                sensorAge.setText(getResources().getQuantityString(R.plurals.sensor_age, sensor_age / 1440, JoH.qs(((double) sensor_age) / 1440, 1)) + age_problem);
```
Comment (tolot27) [inline app/src/main/java/com/eveningoutpost/dexdrip/Home.java:2795]: Did you test this expression?

**Comment by tolot27** [comment]
@k2s Please can you add a description to this PR?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1649,11 +1649,11 @@
...
     <plurals name="sensor_expires">
-        <item quantity="one">Expires: %dd</item>
-        <item quantity="other">Expires: %dd</item>
+        <item quantity="one">Expires: %sd</item>
+        <item quantity="other">Expires: %sd</item>
     </plurals>
     <plurals name="sensor_age">
-        <item quantity="one">Age: %dd</item>
-        <item quantity="other">Age: %dd</item>
+        <item quantity="one">Age: %sd</item>
```
Comment (tolot27) [inline app/src/main/res/values/strings.xml:1656]: Please can you link this format specifiers? What's the difference between `%dd` and `%sd`? I've never seen these double char specfiers.

**Comment by Navid200** [comment]
@k2s Would you please answer the questions?

**Comment by Navid200** [comment]
Can we close this PR please?  If/when the developer has time, we can always reopen it.

**Comment by tolot27** [comment]
> Can we close this PR please? If/when the developer has time, we can always reopen it.

I took this PR over and will finalize it. It is a reasonable improvement.

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java (hunk 1 above)
Comment (tolot27) [reply app/src/main/java/com/eveningoutpost/dexdrip/Home.java:2795]: The int division is necessary for `getQuantityString` to select the correct plural.

**Review by tolot27 — APPROVED** [review]
LGTM and is one step further to get more strings translated.

**Review by jamorham — APPROVED** [review]
No description provided and this does more than change preferences but it looks okay from what I can see and tolot27 has had a good look at it also. The sensor age should be verified in practice to see if it works correctly.

**Pull request merged**